labs/lab-4-jquery/script.js

```diff
@@ -0,0 +1,62 @@
+var party;
+var width;
+// var table;
+
+$(document).ready(function() { // not able to get this to quite work with jQuery
+   $(".form").hide();
+   var resturant = new FloorPlan(3, 3);
+   FloorPlan.render();
+   Table.render();
+});
+
+class Party {
+  constructor(partyName, partySize, partyContact) {
+    this.partyName = partyName;
+    this.partySize = partySize;
+    this.partyContact = partyContact;
+  }
+}
+
+class Table {
+  constructor(tableNum, status, party, available) {
+    this.tableNum = tableNum;
+    this.status = status;
+    this.party = null;
+    this.available = true;
+  }
+
+  reserve(partyName, partySize, partyContact) {
+    if (this.status = true) { // make this happen on click
```

Comment:
What's this if doing here? You've used `=` when I suspect you meant `===`;

But even with a `===`, status starts as undefined ('cause you don't pass in a status on line 54) and only ever gets set to false.